webtau-feature-testing/src/main/groovy/com/twosigma/webtau/featuretesting/WebTauEndToEndTestRunner.groovy

```diff
@@ -129,9 +129,15 @@ class WebTauEndToEndTestRunner implements StepReporter, TestListener {
 
     @Override
     void afterTestRun(WebTauTest test) {
-        scenariosDetails.add([scenario        : test.scenario,
-                              shortContainerId: test.shortContainerId,
-                              stepsSummary    : capturedStepsSummary])
+        def details = [scenario        : test.scenario,
+                       shortContainerId: test.shortContainerId,
+                       stepsSummary    : capturedStepsSummary]
+
+        if (!test.meta.isEmpty()) {
```

Comment:
Why not just add an empty map to details?

Reply:
To avoid multiple broken tests and base line updates. Since meta is not required part of a scenario I figured it will be more convenient to avoid adding it if it is not preset.

Reply:
Ah, yes, I remember having this problem when I changed something about the test output capture json.  Maybe we should revisit how that works at some point but that's for a different day/PR.

Reply:
+1